package.json

```diff
@@ -20,7 +20,8 @@
     "build": "gulp build",
     "build:blocks": "gulp build-blocks",
     "build:core": "gulp build-core",
-    "build:debug": "gulp build-core --verbose > build-debug.log 2>&1 && tail -3 -r build-debug.log",
+    "build:debug": "gulp build-core --verbose --strict",
+    "build:debug:log": "npm run build:debug > build-debug.log 2>&1 && tail -3 -r build-debug.log",
```

Comment:
The -r here makes it not work when I run locally.  Is that a mac vs linux thing?

Reply:
Not sure, removing as it's not necessary.